# REQUIRED - Name of the map
string name

# REQUIRED - Name of EMR parent item
string parent

# Position in relation to parent
geometry_msgs/PoseStamped pose